e2e-tests/protractor.conf.js

```diff
@@ -13,7 +13,7 @@ exports.config = {
 
   baseUrl: 'http://localhost:8000/',
 
-  framework: 'jasmine',
+  framework: 'jasmine2',
```

Comment:
I don't think this is necessary in latest version, but not sure about the version of Protractor we are using here.

Reply:
you're right. Since Protractor v3 it is the same thing.